berkeley-mobile/Common/DetailView/OverviewCardView.swift

```diff
@@ -251,9 +255,25 @@ class OverviewCardView: CardView {
         label.adjustsFontSizeToFitWidth = true
         label.minimumScaleFactor = 0.75
         label.numberOfLines = 0
+        
+        label.isUserInteractionEnabled = true
+//        label.addGestureRecognizer(tap)
+        
         return label
     }()
     
+    @objc func openAddressInMap(_ sender: UITapGestureRecognizer? = nil) {
+//        let myAddress = addressLabel.text
+//        let geoCoder = CLGeocoder()
+//        geoCoder.geocodeAddressString(myAddress!) { (placemarks, error) in
+//            guard let placemarks = placemarks?.first else { return }
+//            let location = placemarks.location?.coordinate ?? CLLocationCoordinate2D()
+//            guard let url = URL(string:"http://maps.apple.com/?daddr=\(location.latitude),\(location.longitude)") else { return }
+//            UIApplication.shared.open(url)
+//        }
+        print("tapped")
+    }
+    
```

Comment:
I know these changes aren't doing anything right now, but could you remove them since they will be completed in a separate PR?

Reply:
I guess Shawn beat me to it :)